aries-site/src/pages/learn/how-to-add-search-and-filter-to-datatable-with-data.mdx

```diff
@@ -0,0 +1,173 @@
+import { Box, Image, Notification } from 'grommet';
+
+## Prerequisites
+
+Let’s get started!
+
+Use this [CodeSandbox template](https://codesandbox.io/s/hpe-design-system-template-lrdtpe?file=/index.js) to start. The template is populated with a paginated DataTable configured to display SpaceX data.
+
+<Notification
+  status="info"
+  width="large"
+  margin={{ bottom: 'medium' }}
+  message="If you’d like to learn more about how to configure a DataTable with a REST API, check out the guide."
+  actions={[
+    {
+      label: 'Guide TBD',
+      href: '#',
+    },
+  ]}
+/>
+
+## Set up Data provider
+
+Start by importing `Data` from grommet.
```

Comment:
Should we specify which file we are working in? e.g. "In`index.js`, start by importing..."

Reply:
Good call.